Packs/SendGrid/pack_metadata.json

```diff
@@ -2,7 +2,7 @@
     "name": "SendGrid",
     "description": "SendGrid provides a cloud-based service that assists businesses with email delivery. It allows companies to track email opens, unsubscribes, bounces, and spam reports. Our SendGrid pack utilize these SendGrid use cases to help you send and manage your emails.",
     "support": "community",
-    "currentVersion": "1.0.2",
+    "currentVersion": "1.1.0",
```

Comment:
This kind of change should not bump the version by 'minor', It should be revision.
Please remove the 1_1_0.md file and execute `demisto-sdk update-release-notes -i Packs/SendGrid -u maintenance`